src/utils/error.ts

```diff
@@ -49,6 +54,22 @@ export abstract class CliKintoneError extends Error {
     }
   }
 
+  private _networkError(error: unknown): error is NetworkError {
```

Comment:
Sorry, I have one more proposal.
The type guard function is better to be named as `isXXX()`.

```suggestion
  private _isNetworkError(error: unknown): error is NetworkError {
```